std/concurrency.d

```diff
@@ -781,32 +747,19 @@ body
 
     Tuple!(T) ret;
 
-    thisInfo.ident.mbox.get(
-        ( T val )
-        {
-            static if ( T.length )
-                ret.field = val;
-        },
-        ( LinkTerminated e )
-        {
-            throw e;
-        },
-        ( OwnerTerminated e )
-        {
-            throw e;
-        },
-        ( Variant val )
-        {
-            static if (T.length > 1)
-                string exp = T.stringof;
-            else
-                string exp = T[0].stringof;
+    thisInfo.ident.mbox.get((T val) {
+        static if (T.length)
+            ret.field = val;
+    }, (LinkTerminated e) { throw e; }, (OwnerTerminated e) { throw e; }, (Variant val) {
```

Comment:
I would use one for each statement, e.g.

```
(LinkTerminated e) { throw e; },
(OwnerTerminated e) { throw e; },
```

but it's already a huge improvement of the status quo!